deployment/glue.yaml

```diff
@@ -433,6 +433,8 @@ Resources:
         "--pii_fields": ""
         "--deleted_fields": ""
         "--dataset_id": ""
+        "--country_code": ""
```

Comment:
It looks like the values we pass here under `DefaultArguments` are what gets used when each key isn't otherwise passed in to the glue job. Do we want to define the default for country_code as "US" here? Or handle that on the API?

Reply:
No users have requested that the API provide a default value for the country code. I recommend that we implement a default some other time.

Reply:
Got it -- I had figured since we wanted to autofill on the frontend but I understand the difference